holmes/plugins/toolsets/mcp/toolset_mcp.py

```diff
@@ -66,7 +64,9 @@ def get_lock_string(self) -> str:
 
 
 @asynccontextmanager
-async def get_initialized_mcp_session(toolset: "RemoteMCPToolset"):
+async def get_initialized_mcp_session(
+    toolset: "RemoteMCPToolset", additional_headers: Optional[Dict[str, str]] = None
```

Comment:
Can you please explain the usage of additional_headers? I don't sure i understand how it is being used, all places either not pass it or put there None. Am i missing something?